rest_tools/server/config.py

```diff
@@ -4,14 +4,25 @@
 import os
 from typing import cast, Dict, Mapping, Optional, Sequence, Union
 
-OptionalDict = Mapping[str, Optional[str]]
-KeySpec = Union[str, Sequence[str], OptionalDict]
 RetVal = Union[str, int, float, bool]
+OptionalDict = Mapping[str, Optional[RetVal]]
+KeySpec = Union[str, Sequence[str], OptionalDict]
```

Comment:
My natural desire to alphabetize wants these in a different order, but that's not a problem with the code, only me. :smile: